setup.py

```diff
@@ -154,6 +154,7 @@
 #     parallel backend to use for intra- and inter-op parallelism
 #     possible values:
 #       OPENMP - use OpenMP for intra-op and native backend for inter-op tasks
+#       NATIVE - use native thread pool for both intra- and inter-op tasks
```

Comment:
Do we eventually want to distinguish between inter and intra or have something like OPENMP_NATIVE and NATIVE_NATIVE to show which one is used for which?

Reply:
My thinking is that we should have:
- OPENMP - OpenMP for intra-op, native thread pool for inter-op
- NATIVE - using native thread pool for both intra- and inter- (actually two separate pools)
- NATIVE_TBB - (next in stack) tbb for intra-op, native thread pool for inter-op
- potentially, just TBB - TBB for both intra- and inter-